pydm/tests/widgets/test_colormaps.py

```diff
@@ -0,0 +1,54 @@
+# Unit Tests for the Frame Widget
+
+
+import pytest
+
+import numpy as np
+
+from ...widgets.colormaps import PyDMColorMap, _magma_data, _inferno_data, _plasma_data, _viridis_data, _jet_data, \
+    _monochrome_data, _hot_data, cmaps, magma, inferno, plasma, viridis, jet, monochrome, hot, cmap_names
+
+
+# --------------------
+# POSITIVE TEST CASES
+# --------------------
+
+def test_construct(qtbot):
+    """
+    Test the construction of the ColorMap widget, and the creations of auxiliary helper objects.
+
+    Expecations:
+    The default values are assigned to the attributes correctly.
+
+    Parameters
+    ----------
+    qtbot : fixture
+        pytest-qt window for widget test
+    """
+    pydm_colormap = PyDMColorMap()
+    qtbot.addWidget(pydm_colormap)
```

Comment:
Colormap is not a Widget... It is just a class used for `ENUMS` with PyQt.

Reply:
Thanks. I've updated the code and the comments.